src/modules/45-dashboards/pages/dashboardView/DashboardView.tsx

```diff
@@ -71,30 +70,20 @@ const DashboardViewPage: React.FC = () => {
     queryParams: { accountId: accountId }
   })
 
-  const links: { url: string; label: string }[] = [
-    {
-      url: routes.toCustomDashboardHome({ accountId }),
-      label: 'Home'
-    },
-    {
-      url: routes.toCustomFolderHome({ accountId }),
-      label: getString('dashboards.homePage.folders')
+  React.useEffect(() => {
+    const links = []
+    if (folderDetail?.resource) {
+      links.push({
+        url: routes.toCustomFolderHome({ accountId }),
+        label: folderDetail?.resource
```

Comment:
You don't need the `?` here... the `if` above asserts it exists
```suggestion
        label: folderDetail.resource
```